sdk/formrecognizer/azure-ai-formrecognizer/azure/ai/formrecognizer/_custom_form_client.py

```diff
@@ -0,0 +1,296 @@
+            ),
+            content_type=content_type,
+            cls=get_pipeline_response,
+            **kwargs
+        )
+
+        def callback(raw_response):
+            model = self._client._deserialize(Model, raw_response)
+            return CustomModel._from_generated(model)
+
+        # FIXME: https://github.com/Azure/azure-sdk-for-python/issues/10417
+        response.http_response.headers["Location"] = response.http_response.headers["Location"] + "?includeKeys=true"
+        return LROPoller(self._client._client, response, callback, LROBasePolling(**kwargs))
+
+    @distributed_trace
+    def begin_labeled_training(self, source, source_prefix_filter="", include_sub_folders=False, **kwargs):
+        # type: (str, Optional[str], Optional[bool], Any) -> LROPoller
+        """Create and train a custom model with labels. The request must include a source parameter that is an
+        externally accessible Azure storage blob container Uri (preferably a Shared Access Signature Uri).
+        Models are trained using documents that are of the following content type - 'application/pdf',
+        'image/jpeg', 'image/png', 'image/tiff'. Other type of content is ignored.
+
+        :param str source: An Azure Storage blob container URI.
+        :param str source_prefix_filter: A case-sensitive prefix string to filter documents in the source path for
+            training. For example, when using a Azure storage blob Uri, use the prefix to restrict sub
+            folders for training.
+        :param bool include_sub_folders: A flag to indicate if sub folders within the set of prefix folders
+            will also need to be included when searching for content to be preprocessed.
+        :return: LROPoller
+        :rtype: ~azure.core.polling.LROPoller
+        :raises: ~azure.core.exceptions.HttpResponseError
+        """
+
+        content_type = kwargs.pop("content_type", "application/json")
+
+        response = self._client.train_custom_model_async(
+            train_request=TrainRequest(
+                source=source,
+                source_filter=TrainSourceFilter(
+                    prefix=source_prefix_filter,
+                    include_sub_folders=include_sub_folders
+                ),
+                use_label_file=True
+            ),
+            content_type=content_type,
+            cls=get_pipeline_response,
+            **kwargs
+        )
+
+        def callback(raw_response):
+            model = self._client._deserialize(Model, raw_response)
+            return CustomLabeledModel._from_generated(model)
+
+        return LROPoller(self._client._client, response, callback, LROBasePolling(**kwargs))
+
+    @distributed_trace
+    def begin_extract_form_pages(self, form, model_id, **kwargs):
+        # type: (Union[str, IO[bytes]], str, Any) -> LROPoller
+        """Analyze Form.
+
+        :param form: .json, .pdf, .jpg, .png or .tiff type file stream.
+        :type form: str or file stream
```

Comment:
could be any stream (even io.BytesIO), so I would do `str or stream`

Reply:
Updated, plan to do a more thorough treatment of docstrings in a later PR